book/C-git-commands/1-git-commands.asc

```diff
@@ -117,162 +118,121 @@ Sa panghuli, ginagamit namin ito upang epektibong ihambing ang mga pagbabago sa
 
-Sa _git_basics_chapter.asc tinatakpan namin ang paggamit ng --amam na opsyon upang gawing muli ang pinakabagong gawa.
+Sa _git_basics_chapter.asc tinalakay namin ang paggamit ng --amend na opsyon upang gawing muli ang pinakabagong commit.
 
-Sa _git_branching.asc, pumunta kami sa mas maraming detalye tungkol sa kung ano ang ginagawa ng git at kung bakit ginagawa nito ito.
+Sa _git_branching.asc, pumunta kami sa mas maraming detalye tungkol sa kung ano ang ginagawa ng git at kung bakit ganito niya ginagawa.
 
-Tiningnan namin kung paano mag-sign magawa ng cryptographically kasama ang `-S` flag sa <<_git_tools#_signing_commits>>.
+Tiningnan namin kung paano mag-sign ng mga commit na cryptographically kasama ang `-S` na palatandaan sa <<_git_tools#_signing_commits>>.
 
-Sa wakas, tinitingnan natin kung ano ang ginagawa ng git commit sa background at kung paano ito aktwal na ipinatupad sa <<_git_internals#_git_commit_objects>>.
+Panghuli, tinitingnan natin kung ano ang ginagawa ng git commit sa background at kung paano nito aktwal na ipinatupad sa <<_git_internals#_git_commit_objects>>.
 
 ==== git reset
 
-Ang utos ng pag-reset ng git ay pangunahing ginagamit upang i-undo ang mga bagay, dahil maaari mong sabihin sa pamamagitan ng pandiwa. Ito ay gumagalaw sa paligid ng HEAD pointer at opsyonal na nagbabago sa index o staging area at maaari ring opsyonal na baguhin ang nagtatrabaho direktoryo kung gagamitin mo - hard. Ang pangwakas na opsyon ay ginagawang posible para sa command na ito na mawala ang iyong trabaho kung ginamit nang hindi tama, kaya tiyaking maunawaan mo ito bago gamitin ito.
+Ang utos na `git reset` ay pangunahing ginagamit upang i-undo ang mga bagay, parang maaari mong sabihin sa pamamagitan ng pandiwa. Ito ay gumagalaw sa paligid ng HEAD pointer at opsyonal na nagbabago sa index o sa lugar ng staging at maaari ring opsyonal na baguhin ang gumaganang direktoryo kung gagamitin mo ang `--hard`. Ang pangwakas na opsyon ay posibleng magbura sa iyong tinatrabaho kung ginamit nang hindi tama, kaya tiyaking maunawaan mo ito bago gamitin.
 
-We first effectively cover the simplest use of `git reset` in <<_git_basics_chapter#_unstaging>>, where we use it to unstage a file we had run `git add` on.
-Una naming epektibong sinasakop ang pinakasimpleng paggamit ng git reset sa _git_basics_chapter.asc, kung saan ginagamit namin ito upang unstage isang file na kami ay tumakbo git idagdag sa.
+Una naming epektibong tinalakay ang pinakasimpleng paggamit ng git reset sa _git_basics_chapter.asc, kung saan ginagamit namin ito upang ma unstage isang file na pinatakbo ang `git add`.
```

Comment:
Please be careful with this tags <<_git_basics_chapter#_unstaging>>
Don't remove them.